website/pages/docs/internals/telemetry.mdx

```diff
@@ -104,6 +104,7 @@ These metrics represent operational aspects of the running Vault instance.
 | `vault.core.step_down`               | Duration of time taken by cluster leadership step downs. This should be monitored and alerted on for overall cluster leadership status.                                                             | ms   | summary |
 | `vault.core.unseal`                  | Duration of time taken by unseal operations                                                                                                                                                         | ms   | summary |
 | `vault.core.unsealed`                | Has value 1 when Vault is unsealed, and 0 when Vault is sealed.                                                                                                                                     | bool | gauge   |
+| `vault.expire.leases.by_expiration`  | Number of leases set to expire, grouped by a time interval. This time interval and total number of time intervals are configurable via `lease_metrics_epsilon` and `num_lease_metrics_buckets` in the telemetry stanza of a vault server configuration. One can additionally group lease expiration by namespace by setting `add_lease_metrics_namespace_labels` to `true` in the config file.                                                                                                                                    | objects  | gauge   |
```

Comment:
A couple suggestions here:

* list the label names here (cluster, whatever you called the bucket) like the other labelled metrics.
* specify the default interval, since that it is what most people will be dealing with
* link to the telemetry stanza documentation?  (Should that be part of this PR?)

There should be a "leases" section, with the other vault.expire metrics, please put it there.

Reply:
"object" can be "leases" to use what is used for vault.expire.num_leases.

Reply:
How does the new change look?